src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template

```diff
@@ -6,7 +6,7 @@ Requires=${{start_facilities}}
 Type=simple
 WorkingDirectory=${{chdir}}
 EnvironmentFile=${{env_config}}
-ExecStart=${{chdir}}/bin/${{exec}}
+ExecStart=${{chdir}}/bin/${{exec}} $RUN_OPTS
```

Comment:
I took a look at the code. Currently only the debian script supports this. Using global variables for application startup seems insecure and magic. 

What is the problem you are trying to solve?